Core/Core/API/APIDiscussionRequestable.swift

```diff
@@ -37,6 +37,7 @@ struct PostDiscussionTopicRequest: APIRequestable {
 }
 
 // https://canvas.instructure.com/doc/api/discussion_topics.html#method.discussion_topics_api.add_entry
+// https://canvas.instructure.com/doc/api/discussion_topics.html#method.discussion_topics_api.add_reply
```

Comment:
Is this line a duplicate of the line above it?

Reply:
Nope. Different anchor, slightly different text. Felt like a waste making another datatype